src/commands/talk.ts

```diff
@@ -0,0 +1,224 @@
+// External dependencies
+import { writeFileSync, createReadStream } from 'node:fs';
+import { isNumber } from 'lodash';
```

Comment:
Lodash is big and expensive to load. Let's not load the whole thing ^^

```suggestion
import isNumber from 'lodash/isNumber';
```